modules/osci/Makefile

```diff
@@ -48,16 +44,19 @@ export OSCI_GIT_USER_NAME ?= "ACM CICD"
 export OSCI_GIT_USER_EMAIL ?= "acm-cicd@redhat.com"
 export OSCI_GIT_MESSAGE ?= Added or Updated $(OSCI_COMPONENT_NAME)
 
+export OSCI_Z_RELEASE_VERSION ?= $(shell curl --silent -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3.raw" https://raw.githubusercontent.com/${OSCI_PIPELINE_ORG}/release/${OSCI_COMPONENT_BRANCH}/Z_RELEASE_VERSION)\
+
+export OSCI_COMPONENT_SUFFIX ?= $(OSCI_COMPONENT_SHA256)
+export OSCI_COMPONENT_TAG ?= $(OSCI_Z_RELEASE_VERSION)-$(OSCI_COMPONENT_SUFFIX)
+
 export OSCI_MANIFEST_QUERY ?= .[] |select(.["image-name"] == "$(OSCI_COMPONENT_NAME)")
-export OSCI_ADDITION_QUERY ?= .[. | length] |= . + {"image-name": "$(OSCI_COMPONENT_NAME)", "image-version": "$(OSCI_COMPONENT_VERSION)", "image-tag": "$(OSCI_COMPONENT_TAG)", "git-sha256": "$(OSCI_COMPONENT_SHA256)", "git-repository": "$(OSCI_COMPONENT_REPO)",  "image-remote": "$(OSCI_IMAGE_REMOTE_REPO_DST)", "image-remote-src": "$(OSCI_IMAGE_REMOTE_REPO_SRC)"}
+export OSCI_ADDITION_QUERY ?= .[. | length] |= . + {"image-name": "$(OSCI_COMPONENT_NAME)", "image-version": "$(OSCI_RELEASE_VERSION)", "image-tag": "$(OSCI_COMPONENT_TAG)", "git-sha256": "$(OSCI_COMPONENT_SHA256)", "git-repository": "$(OSCI_COMPONENT_REPO)",  "image-remote": "$(OSCI_IMAGE_REMOTE_REPO_DST)", "image-remote-src": "$(OSCI_IMAGE_REMOTE_REPO_SRC)"}
```

Comment:
Do you mean for this to be `OSCI_RELEASE_VERSION` or `OSCI_Z_RELEASE_VERSION`, not sure if tag and version are meant to be unique.

Reply:
The `image-version` field is used to form the floating tag in the CI registry that would be used to pull/mirror the component.  This floating tag is always in the `X.Y` format, the value in `OSCI_RELEASE_VERSION`, rather than the `X.Y.Z` in `OSCI_Z_RELEASE_VERSION`.